requirements.txt

```diff
@@ -8,3 +8,5 @@ PyYAML
 paramiko
 cryptography
 setuptools
+requests
+urllib3
```

Comment:
Per the note in requirements.txt these need to be removed.  This should only be the most minimal list of deps for ansible.